pages/login.tsx

```diff
@@ -163,7 +163,7 @@ export default function Login() {
                             Login
                         </button>
                         <div className="mt-1">
-                            <Link href="/">
+                            <Link href="/forgetPassword">
```

Comment:
I think it would be more appropriate to name link this link and the file `resetpassword.tsx`

Reply:
Change the link, file name and end point name in 
https://github.com/nubpro/nakvaksin/pull/5/commits/ddfcbf55bb2d13a3f1bbae292acdd3b381c02ffa
https://github.com/nubpro/nakvaksin/pull/5/commits/aed0e07da53eb35e2b052d6d324243cc49299349